spec/features/users/03_user_signs_in_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+feature "user signs In", %{
+  As unauthenticated user
+  I want to sign in
+  So that I add donut and review the app.
+} do
+  # Acceptance Criteria
+  # [ ] Link to sign in exists in the nav-bar
+  # [ ] Clicking Sign in shows the sign-in form
+  # [ ] Signing in directs us to donuts page
+
+  scenario "user sees sign in" do
```

Comment:
This isn't necessary. We know the sign in link appears because it's part of the following happy path feature test